addons/mail/static/src/models/thread/thread.js

```diff
@@ -1418,9 +1490,19 @@ function factory(dependencies) {
         followers: one2many('mail.follower', {
             inverse: 'followedThread',
         }),
+        futureActivities: one2many('mail.activity', {
+            compute: '_computeFutureActivities',
+            dependencies: ['activitiesState'],
+        }),
```

Comment:
missing comment above filtered activities fields

Reply:
I didn't add those that were missing for those that I just moved, but ok to add them